test/test_crop_coverage.py

```
@@ -0,0 +1,300 @@
+#!/usr/bin/env python3
+"""
+Тест покрытия культур в системе JXCT
+Проверяет, что все культуры правильно отображаются и работают
+"""
+
+import json
```

Ruff (0.12.2)

`json` imported but unused

Remove unused import: `json`

(F401)